Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
> That is when you press build, if your system is 32 bit, you will get that message.

I understand that...  but currently Mac is 32bit and works in 32bit.  We don't want to block building on Mac.